OMCompiler/Compiler/Util/Error.mo

```diff
@@ -1072,7 +1072,7 @@ public constant ErrorTypes.Message NON_STATE_STATESELECT_ALWAYS = ErrorTypes.MES
 public constant ErrorTypes.Message STATE_STATESELECT_NEVER = ErrorTypes.MESSAGE(593, ErrorTypes.SYMBOLIC(), ErrorTypes.WARNING(),
   Gettext.gettext("Variable %s has attribute stateSelect=StateSelect.never, but was selected as a state"));
 public constant ErrorTypes.Message FUNCTION_HIGHER_VARIABILITY_BINDING = ErrorTypes.MESSAGE(594, ErrorTypes.TRANSLATION(), ErrorTypes.WARNING(),
-  Gettext.gettext("Component ‘%s’ of variability %s has binding %s of higher variability %s."));
+  Gettext.gettext("Component '%s’ of variability %s has binding %s of higher variability %s."));
```

Comment:
You missed one.

Also, I thought we were going in the other direction, i.e. replacing ' with ‘ ? But I don't remember why, I think it was @sjoelund who started it?

Reply:
Thanks.

To be honest I don't know if we decided to change them.
I just noticed that in the Jenkins error messages these `‘` don't show up and that I need to google how to type these^^

```
-[flattening/modelica/scodeinst/FunctionInverseInvalid3.mo:15:14-15:43:writable] Error: Invalid inverse annotation for f, only functions with exactly one output may have an inverse.
+[flattening/modelica/scodeinst/FunctionInverseInvalid3.mo:15:14-15:43:writable] Error: Invalid inverse annotation for 'f', only functions with exactly one output may have an inverse.
```

Reply:
I'll update them in another PR

Reply:
Sorry, I seem to have created two discussions, one here and [one in the commit](https://github.com/OpenModelica/OpenModelica/commit/b0c67fe80941f4279c480126872357de1fc6e52a)? My first comment didn't show up so I wrote it again, but I must have accidentally swapped between the PR and the commit somehow.

Anyway, I'll just add what @sjoelund and me wrote there:

> > I think it was @perost who started that :D
> 
> I couldn't imagine that I started it, so I dug through the commit history and the earliest use seems to be #2321 by you. I might have missed something though, it's hard to search for those special characters.
> 
> After that I also started using them, but I don't remember why. But it seems I've actually been using them wrong all this time since I didn't realize that the opening and closing apostrophes you used where different (they look the same in many fonts). I assume that's why @AnHeuermann missed the one above, since it's the only error message where the closing apostrophe is a different character then the opening one.
> 
> But anyway, if there's no actual technical reason for why we used those special apostrophes then I see no reason to use them. They're just annoying to use since you can't write them normally and have to copy-paste them everytime you want to use them.